common-arcana.lic

```diff
@@ -341,4 +348,12 @@ module DRCA
 
     cast?(data['cast'], data['symbiosis'], data['before'], data['after'])
   end
+
+  def check_to_harness(should_harness)
+    return false unless should_harness.equals? 'true'
+    return false unless DRSkill.getxp('Attunement') < DRSkill.getxp('Arcana')
+    return false unless DRSkill.getxp('Attunement') < 30
```

Comment:
Do you think `return false if DRSkill.getxp('Attunement') >= 30` would be clearer?

Reply:
Sure, changed it.